gulp/validateJson.ts

```diff
@@ -48,31 +48,32 @@ validateJsonTask.description =
 
 async function validateFiles(config: ValidationConfig): Promise<void> {
   const fileNames = await glob(config.dataGlob);
-  const validate = ajv.compile(
-    JSON.parse((await fs.readFile(config.schemaPath)).toString())
-  );
+
   const results = await Promise.all(
     fileNames.map(async (fileName) => {
-      const valid = validate(
+      const object = config.schema.safeParse(
         JSON.parse((await fs.readFile(fileName)).toString())
       );
-      if (!valid) logErrors(fileName, validate);
-      return valid;
+      if (!object.success) {
+        // log.error(object.error.flatten());
+        const issues = object.error.issues;
+        issues.forEach((issue) => {
+          const logMsg = `${fileName}: ${issue.path.join(".")} - ${
+            issue.message
+          }`;
+          log(logMsg);
+          log.error(logMsg);
```

Comment:
Do we really need two logs if we log the same message on both?

Reply:
I misread the previous code and thought it was logging the same thing twice. I adjusted the code.